openstack/cronus/values.yaml

```diff
@@ -529,6 +529,85 @@ rhea_rabbitmq:
   metrics:
     enabled: false
 
+maillog:
+  enabled: false
+  server:
+    http: 443
+    maxRequestSize: 52428800
+  storage:
+    type: "elasticsearch"
+    uri: "http://localhost:9200"
+    indexMgmt: false
+    indexPrefix: "mailstate"
+    indexDateFormat: "2006_02_01_15_04_05"
+    cutOffTime: 30d
+    granularity: 1d
+    lookBacks: 5
+  tls:
+    serverTlsName: tls-secret-maillog
```

Comment:
The server tls certs should be handled automagically by disco I believe. So your application shouldn't need to worry about server certs to outside world AFAIK. I.E. the following ingress annotations should take care of updating dns and using server certs in the ingress:
```
    ingress.kubernetes.io/auth-tls-secret: kube-system/ingress-cacrt
    ingress.kubernetes.io/auth-tls-verify-client: "optional"
    ingress.kubernetes.io/auth-tls-verify-depth: "3"
    kubernetes.io/ingress.class: "nginx"
    disco: "true"
    kubernetes.io/tls-acme: "true"
    ```
    This is an example and my not be 100% correct.